addons/event/models/event_event.py

```diff
@@ -786,9 +786,4 @@ def _get_event_print_details(self):
             'timeframe': self._get_event_timeframe_string(),
```

Comment:
Commit message could link the initial task ID to have at least a bit of tracking.